apiserver/facades/agent/upgradesteps/upgradesteps_test.go

```diff
@@ -23,30 +23,33 @@ import (
 type upgradeStepsSuite struct {
 	jujutesting.BaseSuite
 
-	tag names.Tag
-	arg params.Entity
-
 	api        *upgradesteps.UpgradeStepsAPI
 	authorizer *facademocks.MockAuthorizer
 	entity     *mocks.MockEntity
-	machine    *mocks.MockMachine
 	resources  *facademocks.MockResources
 	state      *mocks.MockUpgradeStepsState
 }
 
-var _ = gc.Suite(&upgradeStepsSuite{})
+type machineUpgradeStepsSuite struct {
+	upgradeStepsSuite
+
+	tag     names.Tag
+	arg     params.Entity
+	machine *mocks.MockMachine
+}
 
-func (s *upgradeStepsSuite) SetUpTest(c *gc.C) {
+//var _ = gc.Suite(&upgradeStepsSuite{})
```

Comment:
Should this be commented out?

Reply:
yes, it's in another spot, forgot to remove.  done now.